app-interface/cronjobs/rhose-stage-prolong.yml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: rhose-stage-prolong-cron
+objects:
+  - apiVersion: batch/v1
+    kind: CronJob
+    metadata:
+      name: rhose-stage-prolong-cron
+    spec:
+      schedule: ${SCHEDULE}
+      concurrencyPolicy: Forbid
+      suspend: ${SUSPEND}
+      jobTemplate:
+        metadata:
+          labels:
+            cronjob: rhose-stage-prolong
+        spec:
+          backoffLimit: ${{BACKOFF_LIMIT}}
+          activeDeadlineSeconds: ${{ACTIVE_DEADLINE_SECONDS}}
+          template:
+            metadata:
+              labels:
+                cronjob: rhose-stage-prolong
+            spec:
+              restartPolicy: OnFailure
+              containers:
+                - name: rhose-stage-prolong
+                    image: registry.access.redhat.com/ubi8/ubi
+                    imagePullPolicy: IfNotPresent
+                    command:
+                      - /bin/bash
+                      - -c
+                      - |
+                        # install prerequisites
+                        dnf copr enable ocm/tools
+                        dnf install -y ocm-cli jq
+
+                        ocm login --url staging --token $STAGE_TOKEN
+                        
+                        for CLUSTER_NAME in $(echo $CLUSTER_NAMES | sed "s/,/ /g")
+                        do
+                          echo "Getting id of cluster $CLUSTER_NAME"
+                          CLUSTER_ID=`ocm get /api/clusters_mgmt/v1/clusters --parameter search="name like '$CLUSTER_NAME'" | jq '.items[0].id'`
+                          echo "Prolonging cluster $CLUSTER_NAME with ID $CLUSTER_ID"
+                          printf '{\n\t"expiration_timestamp": "%s"\n}\n' "$(date --iso-8601=seconds -d '+7 days')" | ocm patch /api/clusters_mgmt/v1/clusters/${CLUSTER_ID}
+                        done
+                    env:
+                    - name: CLUSTER_NAMES
+                      value: ${CLUSTER_NAMES}
+                    - name: STAGE_TOKEN
+                      valueFrom:
+                        secretKeyRef:
+                          name: ${STAGE_CRON_JOB_SECRET_NAME}
```

Comment:
As far as I see this assume that we have a single stage cluster. It should be ok and we can always create another similar job if necessary.